### PR TITLE
BO-2050: IG for adding concrete to abstract product

### PR DESCRIPTION
**Pull request by tobi812**
## PR Description

IG for adding concrete to abstract product.

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Comment by ilyakubanov** [comment]
Please add a section With translations generating. (`console translator:generate-cache`, just check in other IGs)

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-core-feature-integration.md [hunk 1]
```diff
@@ -197,8 +197,43 @@ Make sure that the following changes have been applied in transfer objects:
...
+| GuiTableEditableCreateConfiguration | class | Created | src/Generated/Shared/Transfer/GuiTableEditableCreateConfigurationTransfer |
+| GuiTableEditableUpdateConfiguration | class | Created | src/Generated/Shared/Transfer/GuiTableEditableUpdateConfigurationTransfer |
+| GuiTableEditableButton | class | Created | src/Generated/Shared/Transfer/GuiTableEditableButtonTransfer |
+| GuiTableEditableUrl | class | Created | src/Generated/Shared/Transfer/GuiTableEditableUrlTransfer |
+| GuiTableEditableInitialData | class | Created | src/Generated/Shared/Transfer/GuiTableEditableInitialDataTransfer |
+| GuiTableEditableDataError | class | Created | src/Generated/Shared/Transfer/GuiTableEditableDataErrorTransfer |
+| GuiTableDataResponse | class | Created | src/Generated/Shared/Transfer/GuiTableDataResponseTransfer |
+| GuiTableRowDataResponse | class | Created | src/Generated/Shared/Transfer/GuiTableRowDataResponseTransfer |
+| GuiTableDataResponsePayload | class | Created | src/Generated/Shared/Transfer/GuiTableDataResponsePayloadTransfer |
+| GuiTableDataResponsePayload | class | Created | src/Generated/Shared/Transfer/GuiTableDataResponsePayloadTransfer |
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-core-feature-integration.md:224]: the same

File: docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-product-feature-integration.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ To start feature integration, integrate the required features:
 |-|-|-|
 | Marketplace Product | dev-master | [Marketplace Product feature integration](/docs/marketplace/dev/feature-integration-guides/{{page.version}}/marketplace-product-feature-integration.html) |
 | Marketplace Merchant Portal Core | dev-master | [Marketplace Merchant Portal Core feature integration](/docs/marketplace/dev/feature-integration-guides/{{page.version}}/marketplace-merchant-portal-core-feature-integration.html) |
+| Marketplace Inventory Management  | dev-master | [Marketplace Inventory Management feature integration](/docs/marketplace/dev/feature-integration-guides/{{page.version}}/marketplace-inventory-management-feature-integration.html) |
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-product-feature-integration.md:22]: why? it is not listed in recommended dependencies: https://github.com/spryker-feature/marketplace-merchant-portal-product-management 
Please remove and declare all needed transfers in this IG

Comment (tobi812) [reply docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-product-feature-integration.md:22]: In this PR here (https://github.com/spryker/spryker/pull/8421/files#diff-17928f854a728fb780dd37e1b663dee6573f39bacf8a1b7d67f3267bdf8a4cf4) we add a dependency from  the ProductMerchantPortalGui to the MerchantStock module which is part of the inventory-management-feature.

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-core-feature-integration.md (hunk 1 above)
Comment (tobi812) [reply docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-core-feature-integration.md:224]: What do you mean?

File: docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-product-feature-integration.md (hunk 2 above)
Comment (tobi812) [reply docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-product-feature-integration.md:22]: I don't know why it was not added to the feature repository.

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-core-feature-integration.md (hunk 1 above)
Comment (ilyakubanov) [reply docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-core-feature-integration.md:224]: GuiTableDataResponsePayload mentioned 2 times

File: docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-product-feature-integration.md (hunk 2 above)
Comment (ilyakubanov) [reply docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-product-feature-integration.md:22]: ok, lets have it

**Review by ilyakubanov — APPROVED** [review]

**Review by gibmirmilch — APPROVED** [review]

**Pull request merged**